Dockerfile

```diff
@@ -2,23 +2,28 @@ FROM ubuntu:18.04
 
 LABEL maintainer="team@appwrite.io"
 
-RUN apt-get update && \
-    apt-get upgrade -y && \
-    apt-get install -y ruby ruby-dev zlib1g-dev build-essential && \
-    apt-get update && \
-    gem install redis && \
-    gem install resque-web && \
-    gem install resque-scheduler-web && \
-    apt-get autoremove -y ruby-dev zlib1g-dev build-essential && \
-    apt-get clean -y
+EXPOSE 5678
+
+ENV RESQUE_WEB_HOST 127.0.0.1
+ENV RESQUE_WEB_PORT 6379
 
-ADD ./config.ru /config.ru
-ADD ./entrypoint.sh /entrypoint.sh
+COPY ./config.ru /config.ru
+COPY ./entrypoint.sh /entrypoint.sh
 
 RUN chmod 775 /entrypoint.sh
 
-ENV RESQUE_WEB_HOST 127.0.0.1
-ENV RESQUE_WEB_PORT 6379
+RUN apt-get update && \
```

Comment:
I recommend to move this step as a whole to the top right after the `Label` step to avoid invalidating the whole docker cache in case you just need to update any of the copied files for example.

Reply:
Those files use environmental variables to configure the service, I don't think they change that frequently.

Reply:
What was the point of moving them to the bottom anyway? If anyone modified the config files for any reason it could lead to cache invalidation